### PR TITLE
Removes Zipkin from quick start and tracer list

### LINKED ISSUE #257: Swap something else in for Zipkin in the opentracing.io "shell" example
Per @adriancole's request in https://github.com/opentracing/opentracing.io/pull/256#issuecomment-377925846

I am spinning this off into its own issue as it requires some coordination across repositories.

### LINKED ISSUE #258: Take Zipkin off the "supported" list
No one is supporting Zipkin, @pavolloffay writes code in brave-opentracing and doesn't fix bugs. Usage patterns in READMEs are questionable at best and impact us. Your java issues list includes  changes that impact us but don't proceed.

Please take Zipkin off the supported list as it misrepresents it. Doing ones best despite the odds is far from "supported"

cc openzipkin people who have stake in opentracing @basvanbeek @devinsba @jcchavezs

### PR DESCRIPTION
**Pull request by adriancole**
This should help reduce confusion that Zipkin is an OpenTracing project,
that it is a  tracer (it is a system), and any conflation that a single non-contrib
tracer in the OpenZipkin org means we support or are supported by OpenTracing.

Personally, I'm also removing my talk as I am not interested in
indirectly endorsing this project.

Fixes #258
Fixes #257 (though the referenced repo should switch to jeager tracer)

### CODE REVIEW

**Comment by adriancole** [comment]
https://github.com/opentracing/opentracing.io/pull/260 will fix this

**Comment by adriancole** [comment]
changed the example, which for some reason is in a different repo, and different than the quick start here. https://github.com/opentracing-contrib/examples/pull/1

File: _docs/pages/quick-start.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Both of these examples assume that Jaeger is running locally via Docker:
 $ docker run -d -p 5775:5775/udp -p 16686:16686 jaegertracing/all-in-one:latest
 ```
 
-These can be adapted to use Zipkin or any other OpenTracing-compatible Tracer easily by adjusting the initialization code to match the particular implementation.
+These can be adapted to use other OpenTracing-compatible Tracer easily by adjusting the initialization code to match the particular implementation.
```
Comment (JStickler) [inline _docs/pages/quick-start.md]: Subject/verb agreement
"These can be adapted to use other OpenTracing-compatible Tracer**s**"
or
"These can be adapted to use **any** other OpenTracing-compatible Tracer"

Comment (tedsuo) [reply _docs/pages/quick-start.md]: It's ok, we can fix this later.

"fix it in post" :)

**Comment by tedsuo** [comment]
Sorry to see you go @adriancole. Hopefully later, when things are stable, you will find OT instrumentation useful for the Zipkin community. Please let me know if there is anything else I can do for you.

**Comment by adriancole** [comment]
thanks, Ted and great attitude! much appreciated

**Pull request merged**